internal/provider/resource_cluster_test.go

```diff
@@ -52,6 +52,43 @@ func TestAccResourceVcfClusterCreate(t *testing.T) {
 	})
 }
 
+func TestAccResourceVcfClusterStretchUnstretch(t *testing.T) {
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testCheckVcfClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccVcfClusterResourcеStretchTestConfig(
+					"sfo-w02-cl02",
+					""),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("vcf_cluster.cluster", "is_stretched"),
```

Comment:
Maybe it is worth checking if the value is "false", then "true", then "false" again.

Reply:
Good idea, done